task5/src/main/java/com/example/demo111/Status.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo111;
+
+import java.io.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+
+@WebServlet(name = "status", value = "/status")
+public class Status extends HttpServlet {
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.getOutputStream().println("<html><p style=\"color:red\">Application is running </p></html>");
```

Comment:
Consider using constants for immutable string values, as well as for invariable numbers